tests/Fixtures/ConvertConfig/custom_extension.php

```diff
@@ -0,0 +1,81 @@
+<?php
```

Comment:
This file is used in some tests. When we define an extension, we need to have a real extension that can be loaded. Also, if we define a custom formatter, this formatter needs to exist